### PR TITLE
Added new template

### PR DESCRIPTION
**Pull request by Taiga-Tiny**
basic-full-screenshot

### CODE REVIEW

File: basic/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+✅ SEO Friendly
+
+✅ Compatible with any web server
+
+✅ 0 Dependencies
+
+## How to use
+
+1. Close or manually download the code.
+2. Under the "plain" folder, edit the file `config.js`.
```
Comment (jofftiquez) [inline basic/README.md:24]: Please double check all steps if they are applicable and if yes, make sure all content matches with the current template you have.

File: basic/index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Basic Portfolio" />
+    <title>Basic Portfolio</title>
```
Comment (jofftiquez) [inline basic/index.html:9]: Please follow the title syntax in the CONTRIBUTING.md

**Review by jofftiquez — CHANGES_REQUESTED** [review]

**Comment by jofftiquez** [comment]
Thanks for this @Taiga-Tiny please see comments for change requests

File: basic/index.html (hunk 2 above)
Comment (Taiga-Tiny) [reply basic/index.html]: Done guru

File: basic/README.md (hunk 1 above)
Comment (Taiga-Tiny) [reply basic/README.md]: Done guru

**Review by jofftiquez — APPROVED** [review]

**Comment by jofftiquez** [comment]
All good @Taiga-Tiny. One last comment. Please put the latest template in the `index.hml` as well. Here:

image

**Pull request merged**